spec/requests/render_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe 'rendering a post via the API', type: :request do
+  let(:content) { '### Hello, world!' }
+  let(:pipeline) { 'default' }
+  let(:checksum) { Digest::SHA1.hexdigest(content) }
+
+  before do
+    post '/v1/render',
+         params: {
+           checksum: checksum,
+           pipeline: pipeline,
+           content: content
+         }.to_json,
+         headers: { 'Content-Type' => 'application/json' }
+  end
+
+  it 'returns a rendered form of the content' do
+    expect(response.body).to eq('<h3>Hello, world!</h3>')
```

Comment:
Could we return json here just to have a standard response? Pretty much the same as the other endpoint but just a single object?